### PR TITLE
OrthographicTrackballControls updates

### PR DESCRIPTION
**Pull request by patrickfuller**
Addresses #5842.

Incorporated all changes from TrackballControls by diffing through the files. Runs properly on the jsfiddle linked in #5842. @WestLangley This should incorporate all of your recommendations. Thanks for the help!

### CODE REVIEW

File: examples/js/controls/OrthographicTrackballControls.js [hunk 1]
```diff
@@ -250,7 +306,7 @@ THREE.OrthographicTrackballControls = function ( object, domElement ) {
 		if ( !_this.noZoom ) {
 
 			_this.zoomCamera();
-			_this.object.updateProjectionMatrix();
+                        _this.object.updateProjectionMatrix();
```
Comment (mrdoob) [inline examples/js/controls/OrthographicTrackballControls.js]: Spaces indentation here... O:)

Comment (patrickfuller) [inline examples/js/controls/OrthographicTrackballControls.js]: fixed :-D

**Comment by mrdoob** [comment]
Many thanks! ^^

**Comment by klebba** [comment]
@patrickfuller thanks!

**Pull request merged**